nl_NL/LC_MESSAGES/discordbot.po

```diff
@@ -10,27 +10,27 @@ msgstr ""
 
 # Username was given a Nitro Booster Ticket...
 msgid "BOOST_TICKET_SUCCESS"
-msgstr "%s was given a Nitro Booster Ticket, redeemable for a month of Sudomemo Plus and a special trophy. Hooray!"
+msgstr "%s Kreeg een nitro boost ticket voor een maand van sudomemo! Hoera!"
```

Comment:
"Nitro Booster Ticket" should be capitalized like that, the first letter of "Sudomemo" should be capitalized, "Plus" appears to be missing after "Sudomemo".